src/common/Userpic/Userpic.js

```diff
@@ -22,11 +23,22 @@ export const Userpic = forwardRef(({
   children,
   ...rest
 }, ref) => {
+  const propsSrc = user?.avatar ?? src;
   const imgRef = useRef();
-  const [finalSrc, setFinalSrc] = useState(user?.avatar ?? src);
+  const [finalSrc, setFinalSrc] = useState(propsSrc);
   const [imgVisible, setImgVisible] = useState(false);
   const [nameVisible, setNameVisible] = useState(true);
 
+  if (isFF(FF_DEV_1507)) {
+    useEffect(()=>{
+      if (propsSrc !== finalSrc) {
+        setFinalSrc(propsSrc);
+        setImgVisible(false);
```

Comment:
if `onLoad` is not called for cached images this can be a problem. but I think everything is good, approve